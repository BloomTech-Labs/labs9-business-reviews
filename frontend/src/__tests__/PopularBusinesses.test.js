import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import React from 'react';
import PopularBusinessesComponent from '../components/PopularBusinesses';

describe('PopularBusinessesComponent />', () => {
  it('renders', () => {
    shallow(<PopularBusinessesComponent />);
  });
  it('matches snapshot', () => {
    const wrapper = shallow(<PopularBusinessesComponent />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
  it('should contain h1.business__header with text "Popular Businesses', () => {
    let app = shallow(<PopularBusinessesComponent/>);
    expect(app.find(".business__header").length).toBe(1);
    expect(app.find(".business__header").text()).toBe("Popular Businesses");
  });
  it('should contain div with id #Businesses', () => {
    let app = shallow(<PopularBusinessesComponent/>);
    expect(app.find("#Businesses").length).toBe(1);
  });
});
