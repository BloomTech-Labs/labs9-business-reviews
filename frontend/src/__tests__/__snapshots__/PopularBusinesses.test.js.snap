// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PopularBusinessesComponent /> matches snapshot 1`] = `
<styled.div>
  <div
    id="Businesses"
  />
  <h1
    className="business__header"
  >
    Popular Businesses
  </h1>
</styled.div>
`;
